lib/hammer_cli_foreman/hosts/common_update_options.rb

```diff
@@ -98,9 +98,12 @@ def request_params
           params['host']['compute_attributes']['volumes_attributes'] = nested_attributes(option_volume_list)
           params['host']['interfaces_attributes'] = interfaces_attributes
         end
-
-        if options["option_image_id"]
-          compute_resource_id = params['host']['compute_resource_id'] || ::HammerCLIForeman::ComputeResources.get_host_compute_resource_id(params['id'])
+        if options['option_image_id']
+          compute_resource_id = case
```

Comment:
@shiramax, 

There is no variable or regex inside case statement then against which value `when` statements will do matching? 
You need to mention something inside case statement otherwise it will take first value from first when statement.
Please correct me if I am wrong.

Reply:
For readability, it would be good to have simple `if/else/ifelse` as I mentioned.

Reply:
@kgaikwad 
from what I understand it possible to create a switch case statement without the case like follows: 

case
when a < 3
  "Smaller than 3"
end

and I also tested it, and it works fine to me, can you confirm it works for you as well ?